jupyter_server/serverapp.py

```diff
@@ -1561,6 +1569,12 @@ def _default_kernel_spec_manager_class(self):
         ),
     )
 
+    event_bus = Instance(
+        EventBus,
+        allow_none=True,
+        help="An EventBus for emitting structured event data from Jupyter Server and extensions.",
+    )
```

Comment:
Event bus is instantiated down on line 1925. What is the purpose of this statement?

Reply:
This is merely defining the trait type and validation that traitlets should do once the trait is created on line 1925. This doesn't do the actual instantiation.

Reply:
Thanks for explaining.